service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -76,9 +76,15 @@ fi
 # Log in with app-default-credentials
 sudo -u "${JUPYTER_USER}" sh -c "terra auth login --mode=APP_DEFAULT_CREDENTIALS"
```

Comment:
I don't _think_ we need the curly braces here. Does `"$JUPYPTER_USER"` amount to the same thing?

Reply:
i believe it does. i just wanted to keep the style consistent. i can remove it here tho..